boards/airbourne/airbourne_board_config_manager.h

```diff
@@ -0,0 +1,37 @@
+#ifndef AIRBOURNEBOARDCONFIGMANAGER_H
+#define AIRBOURNEBOARDCONFIGMANAGER_H
+
+#include "board_config_manager.h"
+
+namespace rosflight_firmware{
+class ROSflight;
+class AirbourneBoard;
+class AirbourneBoardConfigManager : public BoardConfigManager
+{
+public:
+  AirbourneBoardConfigManager();
+  void init(ROSflight *rf, AirbourneBoard *board);
+  hardware_config_t get_max_config(device_t device) override;
+  ConfigManager::config_response check_config_change(device_t device, hardware_config_t config) override;
```

Comment:
I thought about this, but it can cause some issues with the config_response going out of scope. The compiler will optimize it so that there isn't a copy anyways. https://en.wikipedia.org/wiki/Copy_elision#Return_value_optimization